lib/debezium/converters/timestamp.go

```diff
@@ -0,0 +1,56 @@
+package converters
+
+import (
+	"time"
+
+	"github.com/artie-labs/transfer/lib/typing"
+	"github.com/artie-labs/transfer/lib/typing/ext"
+)
+
+type Timestamp struct{}
+
+func (Timestamp) ToKindDetails() typing.KindDetails {
+	return typing.NewKindDetailsFromTemplate(typing.ETime, ext.DateTimeKindType)
+}
+
+func (Timestamp) Convert(value any) (any, error) {
+	castedValue, err := typing.AssertType[int64](value)
+	if err != nil {
+		return nil, err
+	}
+
+	// Represents the number of milliseconds since the epoch, and does not include timezone information.
+	return ext.NewExtendedTime(time.UnixMilli(castedValue).In(time.UTC), ext.DateTimeKindType, ext.RFC3339Millisecond), nil
```

Comment:
Do we need to call `IsValid()` on the return value from `NewExtendedTime`?

Reply:
I don't think so, if we are able to correctly get `time.Time` and parse the year to be larger than 9999, it still passes `time.Format(...)`

https://go.dev/play/p/tOJR2Ho6EZi